SDVModTest/LEEvents.cs

```diff
@@ -1,11 +1,6 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
-namespace UIInfoSuite
-{
+namespace UIInfoSuite {

```

Comment:
I'm not willing to accept this style. I want braces on their own lines everywhere, please.

Reply:
Could you please add a .editorconfig with the code style you'd like for this project?
Because I see `catch` being on a new line some places and on the same line as `}` other places.

Reply:
I've searched the entire solution and don't see a single case of a catch on the same line as a brace. If it is anywhere I definitely want to fix that so if you do see it please point out the file and line number.